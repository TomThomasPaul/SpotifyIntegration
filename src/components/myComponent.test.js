import {shallow, mount,render} from "enzyme";
import React from "react";
import MyComponent from "./myComponent";
import toJson from 'enzyme-to-json';

const wrapper = shallow(<MyComponent />);

console.log(toJson(wrapper));
console.log("heyy");

it("test sample", ()=>{
    expect(wrapper.find("h1").text()).toContain("THERE");
    expect(toJson(wrapper)).toMatchSnapshot();
    
})

