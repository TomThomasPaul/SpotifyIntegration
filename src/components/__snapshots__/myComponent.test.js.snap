// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test sample 1`] = `
<div>
  <h1>
    HEY THERE! test snapshotsss
  </h1>
  ;
</div>
`;
